src/actions/workspace.js

```diff
@@ -0,0 +1,17 @@
+import ActionTypes from '../action-types';
+
+/**
+ * fullscreenWorkspace - action creator
+ *
+ * @param  {String} windowId
+ * @memberof ActionCreators
+ */
+export function fullscreenWorkspace(fullscreen) {
+  return { type: ActionTypes.FULLSCREEN_WORKSPACE, fullscreen };
+}
+
+/**
+ */
+export function stopEslintFromComplaining() {
```

Comment:
You can stop eslint from complaining by using the ‘default’ keyword on the function above.

Reply:
If I remember correctly, if I use default then I have to do something different in the index.js file.. I just need to figure out how to disable the eslint check here.